MeDrop/MeDrop/Sources/My/View/Extension+MyView.swift

```diff
@@ -0,0 +1,141 @@
+                        .frame(maxWidth: .infinity, alignment: .trailing)
+                        .padding(.vertical, jobInnerPaddingRatio * cardHeight)
+                        .border(width: 2, edges: [.top, .bottom], color: generateCardTextColor(cardColor: info.cardColor))
+                        .padding(.top, fullNameJobGapRatio * cardHeight)
+                    
+                    Text(info.phoneNumber)
+                        .font(.bold(14 * cardRatio))
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .padding(.top, jobPhoneNumberGapRatio * cardHeight)
+                    
+                    Text(info.description)
+                        .font(.bold(14 * cardRatio))
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .padding(.top, phoneNumberDescriptionGapRatio * cardHeight)
+                    
+                    Spacer()
+                    
+                    Button {
+                    } label: {
+                        Text("More")
+                        Image(systemName: "arrow.right")
+                    }
+                    .font(.bold(14 * cardRatio))
+                    .frame(maxWidth: .infinity, maxHeight: cardHeight * moreButtonRatio, alignment: .trailing)
+                }
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+                .padding(.leading, leftPaddingRatio * cardWidth)
+                .padding(.trailing, rightPaddingRatio * cardWidth)
+                
+                Image("VerticalPattern")
+                    .resizable()
+                    .frame(maxWidth: cardWidth * verticalPatternRatio, maxHeight: .infinity)
+            }
+            
+            Image("HorizontalPattern")
+                .resizable()
+                .frame(maxWidth: .infinity, maxHeight: cardHeight * horizontalPatternRatio)
+                .offset(y: -cardHeight * moreButtonRatio)
+        }
+        .frame(width: cardWidth, height: cardHeight)
+        .background(info.cardColor)
+        .foregroundColor(generateCardTextColor(cardColor: info.cardColor))
+        .cornerRadius(10)
+        .padding(.top, .spacing50 * cardRatio)
+        .shadow(color: .black, radius: 5, x: 5, y: 5)
+    }
+    
+    // My Profile에 카드가 없을 때 표시되는 View 입니다.
+    func emptyCarouselView() -> some View {
+        ZStack(alignment: .bottomTrailing) {
+            Button {
+            } label: {
+                Image(systemName: "plus.square")
+            }
+            .font(.regular(60 * cardRatio))
+            .frame(width: cardWidth, height: cardHeight)
+        }
+        .frame(width: cardWidth, height: cardHeight)
+        .background(Color.gray)
+        .foregroundColor(Color.black)
+        .cornerRadius(10)
+        .padding(.top, .spacing50 * cardRatio)
+        .shadow(color: .black, radius: 5, x: 5, y: 5)
+    }
+    
+    // My Profile에 카드가 최대로 만들어졌을 때 표시되는 View 입니다.
```

Comment:
```suggestion
    /// My Profile에 카드가 최대로 만들어졌을 때 표시되는 View 입니다.
```

///를 사용하면 자동완성에서 주석이 표시됩니당 ㅋㅋ